markdown_fs.go

```diff
@@ -69,6 +69,9 @@ func newMarkdownFS(p string) *markdownFS {
 
 						name := strings.TrimSuffix(event.Name, ".md")
 						name, _ = filepath.Rel(m.path, name)
+	                    cp := m._page(name)
```

Comment:
I noticed the indentation is not consistent here. I see in my editor that it uses spaces and other lines uses tabs. maybe editor config needs adjusting. I guess `go fmt` needs to run on this file?